anitya/tests/lib/versions/test_calver.py

```diff
@@ -488,8 +488,9 @@ def test_lt_both_prerelease_one_unversioned(self):
 
     def test_lt_nonsense(self):
         """Assert CalendarVersion supports < comparison."""
+        # unclear if it should really be valid -junk modifier?
         old_version = calver.CalendarVersion(
-            version="2019.04.23junk", pattern="YYYY.0M.DD-MODIFIER"
```

Comment:
This was correct, look at the name of the test. I'm trying to test if nonsense version would be lesser than the correct version.

Reply:
Fixed it back to the original.